packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -0,0 +1,3907 @@
+import { getDisplayUrlMatcher } from '@packages/driver/src/cy/net-stubbing/route-matcher-log'
```

Comment:
This uses .cy now; I imagine we can copy forward Ryans changes directly: https://github.com/cypress-io/cypress/commit/9db7d962b27e97404cc2f31c7490b6be94d99168